### DISCUSSION TITLE
Where is the best place to share ideas for possible mods?

### DISCUSSION THREAD

**Discussion by mabkenar** [rgerganov/gen5fw#4] [Ideas]
I have already compiled a detailed list of features and improvements that I'd like to see in Std-Gen5 (well, basically in any IVI that I may use).

Where would be the best place to share them, in your opinion? Having a space for sharing such ideas can be good in different ways.

* I can imagine that some people might be interested in developing modified versions of Std-Gen5, but they lack ideas, and don't know where to start from.
* Such discussions can inspire people who do have ideas, and help them improve their designs based on user discussions and knowing about different use-cases.
* If developers from Hyundai Mobis read the discussion, they may even find some of the features interesting and, even lacking an official user feedback program from Hyundai side, these developers may lobby the feature suggestions into future official Hyundai releases. (I know, less fun for independent developers, but practical for end-users ;)

Would it be ok if I make a new discussion thread here for each new feature that I have in my list? CC @rgerganov

**Comment by rgerganov** [comment]
> Would it be ok if I make a new discussion thread here for each new feature that I have in my list? CC @rgerganov

Sure, go ahead. I think Discussions is a good fit for this.
